lib/puppet-strings/tasks/validate.rb

```diff
@@ -34,7 +34,7 @@
       existing = File.read(filename)
 
       if generated != existing
-        warn "#{filename} is outdated"
+        warn "#{filename} is outdated; to regenerate: bundle exec rake strings:generate:reference"
```

Comment:
I am wondering how many people might call puppet strings not via bundler 🤔 but I am +1 for the change